app/controllers/tracks/exercises_controller.rb

```diff
@@ -29,7 +29,17 @@ def tooltip
 
   def start
     Solution::Create.(current_user, @exercise)
-    redirect_to action: :edit
+
+    respond_to do |format|
```

Comment:
Didn't Jeremy create a shorthand method for this bit of code?

Reply:
I created a shorthand just for the tooltip/modal rendering. This is normal (idiomatic) Rails for the other scenarios.